rest/model/task.go

```diff
@@ -470,6 +471,7 @@ func (ad *APITask) ToService() (interface{}, error) {
 	for i, dep := range ad.DependsOn {
 		dependsOn[i].TaskId = dep.TaskId
 		dependsOn[i].Status = dep.Status
+		dependsOn[i].Unattainable = dep.Unattainable
```

Comment:
Since this is business logic, I hesitate slightly to expose this as a field to rest routes. Any chance of moving this to a resovler?

Reply:
Ahh sorry I didn’t realize that, I removed it. Because I lose access to the `Unattainable` field but don't want to do an additional db lookup, I now use the `DisplayStatus` to check if the task is blocked instead